app/assets/stylesheets/application.scss

```diff
@@ -78,16 +78,16 @@ body {
   background: #EE9D1C;
 }
 
-.btn-game-search {
-  color: rgb(255, 255, 255)!important;
-  background-color: black!important;
-  border-color: black!important;
+#btn-game-search {
+  color: rgb(255, 255, 255);
+  background-color: black;
+  border-color: black;
```

Comment:
FE Note:

!important is used to override bootstrap. for future knowledge when we work with bigger code bases, it might impact our code and we get a lot of errors. also !important is a sign of code smell and usually deemed bad code for FE. Important thing to note in case you work with FE in the future.

Reply:
Good to know.